specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -511,18 +667,12 @@ def verify_cell_kzg_proof_batch(row_commitments_bytes: Sequence[Bytes48],
                                 proofs_bytes: Sequence[Bytes48]) -> bool:
     """
     Verify a set of cells, given their corresponding proofs and their coordinates (row_index, column_index) in the blob
-    matrix. The list of all commitments is also provided in row_commitments_bytes.
-
-    This function implements the naive algorithm of checking every cell
-    individually; an efficient algorithm can be found here:
-    https://ethresear.ch/t/a-universal-verification-equation-for-data-availability-sampling/13240
-
-    This implementation does not require randomness, but for the algorithm that
-    requires it, `RANDOM_CHALLENGE_KZG_CELL_BATCH_DOMAIN` should be used to compute
-    the challenge value.
+    matrix. The i-th cell is in row row_indices[i] and in column column_indices[i].
```

Comment:
This sentence was a little confusing at first. I think some `=` signs here might help.
```suggestion
    matrix. The i-th cell is in row=row_indices[i] and in column=column_indices[i].
```